### ISSUE TITLE
Added "ringmode" command to set/get RingerMode

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#69]
```
Trival enhancement.
Could have needed this command this morning: My phone was lost in my backpack, 
the course had already begun, laptop in front of me and I was not sure if the 
phone was silent or not. Now I can check and set the mode. Yeah! :-)

@Florent: I talked to you about adding new commands outside the MainService 
Class. This bundle contains a proposal from me. As always comments and 
suggestions welcome.
```

Original issue reported on code.google.com by `fschm...@gmail.com` on 1 Feb 2011 at 7:41

Attachments:
- [ringmode.bundle](https://storage.googleapis.com/google-code-attachments/gtalksms/issue-69/comment-0/ringmode.bundle)

**Comment by GoogleCodeExporter** [comment]
```
Oki I've start refactoring based on your modifications.
I'll commit tonight with all commands splited.
```

Original comment by `Florent....@gmail.com` on 1 Feb 2011 at 9:03

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 2 Feb 2011 at 12:50
- Changed state: **WaitingForRelease**

**Comment by GoogleCodeExporter** [comment]
```
Nice, now MainService is down to 700 lines.
2 suggestions in attached bundle:
- changed ringmode back to the old behavior, this way you get feedback about 
the current active ringmode right away.
- explicitly check before sending xhtml messages

Also, I discovered that my xmpp client (gajim) wont show any xhtml formating 
including newlines. I suspect this to be a client problem as the xmpp xml 
message looks right. Will take a close look at this later.
```

Original comment by `fschm...@gmail.com` on 2 Feb 2011 at 8:04

Attachments:
- [ringmode-xhtml.bundle](https://storage.googleapis.com/google-code-attachments/gtalksms/issue-69/comment-3/ringmode-xhtml.bundle)

**Comment by GoogleCodeExporter** [comment]
```
I was wrong, the xhtml tags were not within the specs. Also I disabled the 
check for xhtml support, as it didn't work correctly.


Here is the bundle.
```

Original comment by `fschm...@gmail.com` on 2 Feb 2011 at 6:02

Attachments:
- [xhtmltags.bundle](https://storage.googleapis.com/google-code-attachments/gtalksms/issue-69/comment-4/xhtmltags.bundle)

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v1.9
```

Original comment by `Florent....@gmail.com` on 6 Feb 2011 at 8:27
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 25 Aug 2011 at 7:51
- Changed state: **Deployed**
